goreleaser/darwin.yaml

```diff
@@ -18,6 +18,7 @@ builds:
       - arm64
     ldflags:
       - -s -w
+      - -extldflags "-static"
```

Comment:
Should we do it only for Linux? Looks like it’s only an issue there

Reply:
I don't think it can hurt to keep the builds consistent.